### PR TITLE
EREGCSC-759 - Skip to Main Content

### PR DESCRIPTION
**Pull request by PhilR8**

### CODE REVIEW

File: e2e/package.json [hunk 1]
```diff
@@ -11,7 +11,8 @@
   "devDependencies": {
     "@testing-library/cypress": "^7.0.5",
     "axe-core": "^4.3.0",
-    "cypress": "^6.9.1"
+    "cypress": "^6.9.1",
+    "cypress-plugin-tab": "^1.0.5"
```
Comment (PhilR8) [inline e2e/package.json:15]: Cypress maintainers feel adding tab press functionality is out of scope, so added plugin to enable tab navigation.  See: https://github.com/cypress-io/cypress/issues/299

File: regulations/static/package.json [hunk 2]
```diff
@@ -30,6 +30,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.13.16",
+    "@rollup/plugin-alias": "^3.1.8",
```
Comment (PhilR8) [inline regulations/static/package.json:33]: rollup alias plugin added when fixing main.js lint/formatting errors - now we don't have to import from `../../../../../plugin` locations

File: regulations/static/regulations/js/main.js [hunk 3]
```diff
@@ -1,16 +1,18 @@
-import RelatedRules from "./RelatedRules.js";
-import Collapsible from "./Collapsible.js";
-import CollapseButton from "./CollapseButton.js";
-import SupplementalContent from "./SupplementalContent.js";
-import CopyBtn from "./CopyBtn.js";
-import TableComponent from "./TableComponent.js";
-import Vue from "../../node_modules/vue/dist/vue.esm.browser.min.js";
-import { goToVersion } from "./go-to-version.js";
+import Vue from "vue/dist/vue.esm.browser.min";
```
Comment (PhilR8) [inline regulations/static/regulations/js/main.js:1]: everything in this file was changed to satisfy linter

**Review by knollfear — APPROVED** [review]
LGTM.  Worked as expected after I set my mac up to handle tab nav correctly.

**Pull request merged**